Src/Extern/HYPRE/AMReX_HABEC_3D.F90

```diff
@@ -299,13 +301,13 @@ subroutine hbvec3(vec, &
   else
      print *, "hbvec3: impossible face orientation"
   endif
-end subroutine hbvec3
 
+end subroutine hbvec3
 
 subroutine hmac(mat, a, &
-                DIMS(abox), &
-                DIMS(reg), &
-                alpha) bind(C, name="amrex_hmac")
+     DIMS(abox), &
+     DIMS(reg), &
+     alpha) bind(C, name="amrex_hmac")
```

Comment:
If possible, could we get rid of those white space only changes?  There are a lot of white space changes in several files, not just here.